src/platforms/php/guides/laravel/other-versions/laravel4.mdx

```diff
@@ -6,16 +6,22 @@ redirect_from:
   - /platforms/php/guides/laravel/configuration/other-versions/laravel4/
 ---
 
-Install the `sentry/sentry-laravel` package:
-
 <Note>
 
-Laravel 4.x is supported until version 0.8.x of the `sentry/sentry-laravel` package.
+Laravel 4.x is supported until version 0.8.x of the `sentry/sentry-laravel` package and is no longer actively supported.
```

Comment:
What do you mean by it's not actively supported but it is supported?

Reply:
It is supported in a sense that it still works, but there are no updates/bugfixes anymore.

Suggestions? Maybe is just `and is no longer supported` clearer?

Reply:
I have just changed it to:

```Laravel 4.x is supported until version 0.8.x of the `sentry/sentry-laravel` package.```

If you have a better suggestion I'm all ears :)

Reply:
All good now :)